src/app/resources/resources.component.html

```diff
@@ -25,7 +25,7 @@
     <span class="toolbar-fill"></span>
     <span *ngIf="!parent">
       <button mat-button [disabled]="!selection.selected.length" (click)="addToLibrary(selection.selected)">
-        <mat-icon aria-hidden="true" class="margin-lr-3">library_add</mat-icon><span i18n>Add to Library</span>
+        <mat-icon aria-hidden="true" class="margin-lr-3">library_add</mat-icon><span i18n>Add to myLibrary</span>
```

Comment:
Consistent user messages on camelcase is not quiet userfriendly please refer https://github.com/open-learning-exchange/planet/pull/740/commits/81537c66a05e0ad8f68270becb4a4067005c884c